torchvision/transforms/functional.py

```diff
@@ -948,12 +949,48 @@ def _get_inverse_affine_matrix(
     return matrix
 
 
+def _get_inverse_affine_matrix_tensor(
+    center: Tensor, angle: Tensor, translate: Tensor, scale: Tensor, shear: Tensor
+) -> Tensor:
+    output = torch.zeros(3, 3)
+
+    rot = angle * torch.pi / 180.0
+    shear_rad = shear * torch.pi / 180.0
+
+    m_center = torch.eye(3, 3)
+    m_center[:2, 2] = center
+
+    i_m_center = torch.eye(3, 3)
+    i_m_center[:2, 2] = -center
+
+    i_m_translate = torch.eye(3, 3)
+    i_m_translate[:2, 2] = -translate
+
+    # RSS without scaling
+    sx, sy = shear_rad[0], shear_rad[1]
+    a = torch.cos(rot - sy) / torch.cos(sy)
+    b = torch.cos(rot - sy) * torch.tan(sx) / torch.cos(sy) + torch.sin(rot)
+    c = -torch.sin(rot - sy) / torch.cos(sy)
+    d = -torch.sin(rot - sy) * torch.tan(sx) / torch.cos(sy) + torch.cos(rot)
+
+    output[0, 0] = d
+    output[0, 1] = b
+    output[1, 0] = c
+    output[1, 1] = a
+    output = output / scale
+    output[2, 2] = 1.0
+
+    output = torch.chain_matmul(m_center, output, i_m_center, i_m_translate)
```

Comment:
I think this `chain_matmul` is going to get deprecated soon in core PyTorch in favor of some similar method in `torch.linalg`

Reply:
Thanks for pointing out that ! We have to use `torch.linalg.multi_dot` instead: https://pytorch.org/docs/stable/generated/torch.chain_matmul.html